app/Resources/TwigBundle/views/Exception/error404.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends '::pagelayout.html.twig' %}
+
+{% block main %}
+    <main class="error-page-background">
+{% endblock %}
+
+{% block content %}
+    <div class="content-view-full content-error-page">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-7 content">
+                    <h1>Whoops!</h1>
+                    <h2>We couldn't find the page you were looking for.</h2>
+                    <a href="/" class="btn btn-primary goback">Go back</a>
```

Comment:
or maybe would be better to rename it to `go home` since there is only one page at this moment?
ping @SylvainGuittard